.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+name: Java CI build and test
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, windows-latest, macOS-latest]
+        java: [16-ea]
```

We can drop the `-ea` now. 🥳 

```suggestion
        java: [16]
```

---

Maybe later, if you remove the target source in the pom.xml we could make the build go against 17ea. That way you can take advantage of the java:[] matrix in gha. java:[16,17ea].